voice/bxml/about.md

```diff
@@ -53,15 +55,15 @@ BXML callbacks are HTTP messages that are sent to your application server to not
 | [Gather](callbacks/gather.md)                               | Bandwidth API sends this when a `<Gather>` is completed.                                                                                                                                   |
-| [Initiate](callbacks/initiate.md)                           | Bandwidth API sends this when an inbound call arrives.                                                                                                                                     |
-| [Record Complete](callbacks/recordComplete.md)              | Bandwidth API sends this when a `<Record>` is completed. The recording is not available to download until the [Recording Available](callbacks/recordingAvailable.md) event is received.    |
-| [Redirect](callbacks/redirect.md)                           | Bandwidth API sends this when a `<Redirect>` is requested or when a POST request is made to the [/calls/{callId}](../methods/calls/postCallsCallId.md) endpoint requesting a redirect.     |
-| [Transfer Answer](callbacks/transferAnswer.md)              | Bandwidth API sends this when a `<Transfer>` is answered.                                                                                                                                  |
-| [Transfer Complete](callbacks/transferComplete.md)          | Bandwidth API sends this when the `<Transfer>` is complete and the original call needs to continue.                                                                                        |
+| [Initiate](callbacks/initiate.md)                           | Bandwidth API sends this to the application when an inbound call arrives.                                                                                                                                     |
+| [Record Complete](callbacks/recordComplete.md)              | Bandwidth API sends this to the application when a `<Record>` is completed. The recording is not available to download until the [Recording Available](callbacks/recordingAvailable.md) event is received.    |
+| [Redirect](callbacks/redirect.md)                           | Bandwidth API sends this to the application when a `<Redirect>` is requested or when a POST request is made to the [/calls/{callId}](../methods/calls/postCallsCallId.md) endpoint requesting a redirect.     |
+| [Transfer Answer](callbacks/transferAnswer.md)              | Bandwidth API sends this to the application when a `<Transfer>` is answered.                                                                                                                                  |
+| [Transfer Complete](callbacks/transferComplete.md)          | Bandwidth API sends this to the application when a `<Transfer>` is complete and the original call needs to continue.                                                                                        |
```

Comment:
Synced the text in the about pages.

Reply:
I prefer the version with less text but :shrug:

Reply:
re-synced about files.